proposal/comp110_architecture_proposal.tex

```diff
@@ -14,61 +14,56 @@
+\item[Web link:] \url{http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=5593333}
+\item[Full text link:] \url{https://games.soe.ucsc.edu/sites/default/files/cig10_043CP2_115.pdf}
+\item[Comments:] I found this paper in the IEEE Xplore database. It is published by IEEE so is from a reputable publisher. It is relevant to the essay as it describes a technique for procedural level generation in platform games that requires more input from a human than some of the other techniques.
 \end{description}
 
 \section*{Paper 3}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] Automatic Level Generation for Platform Videogames Using Genetic Algorithms
+\item[Citation:] \cite{mourato:genetic}
+\item[Abstract:] In this document we present an investigation on automatically generating levels for platform videogames. Common approaches for this problem are rhythm based, where input patterns are transformed in a valid geometry, and chunk based, where samples are humanly created and automatically assembled like a puzzle. The proposal hereby presented is to explore this challenge with the usage of Genetic Algorithms, facing it as a search problem, in order to achieve higher expressivity and less linearity than in rhythm based approach and without requiring human creation as it happens with the chunk based approach. With simple heuristics the system is able to generate playable levels in a small amount of time (one level is created in less than a minute) and with considerable diversity, as our results show.
+\item[Web link:] \url{http://dl.acm.org/citation.cfm?id=2071423.2071433}
+\item[Full text link:] \url{}
+\item[Comments:] I found this paper in the ACM Digital Library. It presents an approach to procedural content generation for platform games that is different to those presented in the above papers \cite{smith:launchpad, mawhorter:occupancy}. It is published by ACM, so it is from a reputable publisher.
 \end{description}
 
 \section*{Paper 4}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] A procedural procedural level generator generator
```

Comment:
This is a bit too meta- for me! Interesting stuff, and worth mentioning. However, I would focus elsewhere to ensure that the brief is clearly satisfied.